Index: app/src/main/java/com/example/heiroghliphics_translate_project/models/Addnewfoldermodel.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.heiroghliphics_translate_project.models;\n\npublic class Addnewfoldermodel {\n    private String foldername;\n    private String date;\n    private String image;\n\n    public Addnewfoldermodel() {\n    }\n\n    public String getFoldername() {\n        return foldername;\n    }\n\n    public void setFoldername(String foldername) {\n        this.foldername = foldername;\n    }\n\n    public String getDate() {\n        return date;\n    }\n\n    public void setDate(String date) {\n        this.date = date;\n    }\n\n    public String getImage() {\n        return image;\n    }\n\n    public void setImage(String image) {\n        this.image = image;\n    }\n}\n\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/heiroghliphics_translate_project/models/Addnewfoldermodel.java	(revision c8ab0f7734bbaeef6f6daa240c9bb8826dff810f)
+++ app/src/main/java/com/example/heiroghliphics_translate_project/models/Addnewfoldermodel.java	(date 1619107149000)
@@ -5,6 +5,7 @@
     private String date;
     private String image;
 
+
     public Addnewfoldermodel() {
     }
 
